server/README.md

```diff
@@ -0,0 +1,139 @@
+[![banner](https://raw.githubusercontent.com/oceanprotocol/art/master/github/repo-banner%402x.png)](https://oceanprotocol.com)
+
+<h1 align="center">Commons: Server</h1>
+
+## Introduction
+
+This folder contains server component written in TypeScript using [Express](https://expressjs.com). The server provides various microservices, like remote file checking, retiring and updating metadata.
+
+To spin up the server in a watch mode for local development, execute:
+
+```bash
+npm install
+npm start
+```
+
+## ✨ API Documentation
+
+### 1. Url Checker
+
+Url Checker returns if file exists, size and additional information about requested file. This service is used as a solution to frontend CORS restrictions.
+
+**Endpoint:** POST `/api/v1/urlcheck`
+
+**Input Parameters:**
+
+```json
+{
+    "url": "https://oceanprotocol.com/tech-whitepaper.pdf"
+}
+```
+
+**Return Value**
+
+```json
+{
+    "status": "success",
+    "result": {
+        "found": true,
+        "contentLength": "2989228",
```

Comment:
we return a `number` here, right?

Reply:
We currently return string.

Reply:
ah interesting, cause in client we always assume a number but it still seems to work https://github.com/oceanprotocol/commons/blob/master/client/src/routes/Publish/Files/index.tsx#L16

Reply:
Could be that json parser on client side considers this a number. Will change this on server side to number.

Reply:
Fixed in bdcf2f2